test/EnvironmentManagerTests/EnvironmentVariableHelper.h

```diff
@@ -165,7 +165,7 @@ inline std::wstring GetEnvironmentVariableForProcess(const std::wstring variable
     return environmentVariableValue;
 }
 
-inline void InjectIntoPath(bool isProcess, bool isUser, const std::wstring& pathPart, int index)
+inline void InjectIntoPath(bool isProcess, bool isUser, const std::wstring& pathPart, UINT index)
```

Comment:
`UINT`? How very retro :P

`UINT32` or `uint32_t` is more hip these days

same comment applies throughout

Reply:
I don't want to be retro.